### ISSUE TITLE
phonotactic profiles

### ISSUE THREAD

**Issue by martino-vic** [martino-vic/loanpy#19]
At the moment I'm calculating the phonotactic profiles of words from within loanpy and was wondering if that is not also a data wrangling task that should be moved to the preprocessing? If yes then I could fix this step simultaneously with issue #18 @LinguList

**Comment by LinguList** [comment]
Good point, and yes.

**Comment by LinguList** [comment]
We have one example database, where we added CV templates. 

* https://github.com/lexibank/tppsr/
* https://tpps.clld.org

**Comment by LinguList** [comment]
The question is: what kind of CV templates do you want to calculate?

**Comment by LinguList** [comment]
I mean "prosodic templates"? In lingpy, we have some of them, including those that automatically syllabify.

**Comment by martino-vic** [comment]
Ah, great, so I simply use `from lingpy import prosodic_string` then `ProsodicStructure = attr.ib(default=None)` and finally `ProsodicStructure=prosodic_string(tokens, _output='CcV')`. I think I should be able to implement this 🖒 Actually this should allow me to make the whole code a good deal simpler, could throw out 2 functions and probably even 2-3 files...

**Comment by martino-vic** [comment]
I'm wonderning what the lower case "c" means in this context, I tried to look it up in lingpy's [documentation](https://lingpy.readthedocs.io/en/latest/reference/lingpy.sequence.html#lingpy.sequence.sound_classes.prosodic_string) but couldn't find an answer there. Like for example in

```python
>>> rom lingpy import prosodic_string
>>> prosodic_string(["a", "b", "c"], _output="CcV")
'Vcc'
```

**Comment by LinguList** [comment]
That is indeed nice! Let me know once you have implemented this, and 
I'll have a look and check (for curiosity ;)

**Comment by martino-vic** [comment]
Yup, will do, working on it right now. I figured out in the meanwhile that I'll have to use `_output="cv"` to get only capital C and V... [and T](https://github.com/lingpy/lingpy/blob/master/src/lingpy/sequence/sound_classes.py#L983). So I'm just curious: What does the "T" signify in a prosodic string?

**Comment by martino-vic** [comment]
https://github.com/martino-vic/ronatasbertawot/blob/main/cldf/forms.csv

> Let me know once you have implemented this, and I'll have a look and check (for curiosity ;)

added col ProsodicStructure to [forms.csv](https://github.com/martino-vic/ronatasbertawot/blob/main/cldf/forms.csv) with a modified lexibank-script. Went surprisingly quickly to implement! Took me like less than 5 minutes , eventually

**Comment by martino-vic** [comment]
Ah and lingpy is treating "j" as a consonant, thank god! PanPhon encodes it as a vowel, so I had to manually tinker around in the feature vector file and then document the changes I made - not very elegant

**Comment by martino-vic** [comment]
Fixed this in commit ca23d2868a117ae3ad03b242955273e085c9f959

**Comment by LinguList** [comment]
Please test with strings like `torte` vs. `trote`. Lower c indicates a 
consonant following a syllable peak, so it is post-nucleus.
